python/lsst/qa/explorer/writeParquet.py

```diff
@@ -0,0 +1,107 @@
+from lsst.daf.persistence.butler import Butler
+from lsst.pex.config import (Config, Field, ConfigField, ListField, DictField, ConfigDictField,
+                             ConfigurableField)
+from lsst.pipe.base import Task, CmdLineTask, ArgumentParser, TaskRunner, TaskError
+from lsst.coadd.utils import TractDataIdContainer
+from lsst.pipe.tasks.multiBand import MergeSourcesTask, MergeSourcesConfig
+from lsst.pipe.tasks.multiBand import _makeGetSchemaCatalogs
+from lsst.coadd.utils.coaddDataIdContainer import ExistingCoaddDataIdContainer
+
+
+import functools
+import re
+import pandas as pd
+
+from .table import ParquetTable
+
+
+class WriteObjectTableConfig(MergeSourcesConfig):
+    priorityList = ListField(dtype=str, default=['HSC-G', 'HSC-R', 'HSC-I', 'HSC-Z', 'HSC-Y'],
+                             doc="Priority-ordered list of bands for the merge.")    
+    engine = Field(dtype=str, default="pyarrow", doc="Parquet engine for writing (pyarrow or fastparquet)")
+
+class WriteObjectTableTask(MergeSourcesTask):
```

Comment:
Should this be called `WriteOjbectTableAsParquetTask` or `WriteParquetTableTask`, just to be more explicit?

Reply:
I'm sticking with `WriteObjectTableTask`.  Users shouldn't care that it's parquet if we do our job right.